lms/djangoapps/courseware/access.py

```diff
@@ -160,11 +181,18 @@ def _can_access_descriptor_with_start_date(user, descriptor, course_key):  # pyl
             descriptor,
             course_key=course_key
         )
-        return (
-            descriptor.start is None
-            or now > effective_start
-            or in_preview_mode()
-        )
+        if descriptor.start is None or now > effective_start or in_preview_mode():
+            return ACCESS_GRANTED
+
+        start_error_message = None
+        if hasattr(descriptor, 'advertised_start'):
+            if descriptor.advertised_start is not None:
+                start_error_message = _(descriptor.advertised_start)  # pylint: disable=translation-of-non-string
+
+            elif descriptor.start != DEFAULT_START_DATE:
+                start_error_message = defaultfilters.date(descriptor.start, "DATE_FORMAT")
```

Comment:
Change "start_error_message" to "start".
This is the value for just the start date, not a full error message, right?